content/en/developers/metrics/custom_metrics.md

```diff
@@ -3,226 +3,244 @@ title: Custom Metrics
+| Metric        | Tags                           |
+|---------------|--------------------------------|
+| `temperature` | `state:fl`, `region:southeast` |
+| `temperature` | `state:ma`, `region:northeast` |
+| `temperature` | `state:ny`, `region:northeast` |
+| `temperature` | `state:wa`, `region:northwest` |
 
-Suppose you are interested in measuring the maximum `age` metric in the state of New York. `age` is submitted to Datadog as a distribution metric tagged with `city` and `state` :
+### Distributions
 
-|  | Values in 10s flush interval | Sum | Count | Minimum | Maximum | Average (Sum/Count) |
-|---------------|------------------------------|-----|-------|---------|---------|---------------------|
-| Rochester, NY | 23,29,33,55,41,36,12,67 | 296 | 8 | 12 | 67 | 37 |
-| New York, NY | 18,22,26,31,29,40,23,35 | 215 | 8 | 18 | 40 | 28 |
+A distribution metric gathers all values across all hosts emitting metric values in ten-second flush intervals. Distributions emit a number of custom metrics that is proportional to the number of custom metrics emitted from `GAUGE`. Distributions generate five timeseries for each unique tag value combination that appears in the data: `sum`, `count`, `min`, `max`, and `avg` (calculated from the sum/count).
 
-The total number of custom metrics or timeseries emitted from the `age` distribution metric is **ten (5 x 2)**. For both unique tag value combinations above (Rochester, NY and New York, NY), Datadog stores five timeseries (`sum`,`count`,`min`,`max`, `avg`). 
+Suppose you are interested in measuring the maximum `age` metric in the state of New York. `age` is submitted as a distribution metric tagged with `city` and `state`:
 
-To obtain the maximum `age` in the state of New York, you can reaggregate the timeseries above: Maximum age in New York = `max`(`max`(Rochester, NY), `max`(New York, NY)) = 67.
+| Tags          | Values in 10s flush interval | Sum | Count | Minimum | Maximum | Average (Sum/Count) |
+|---------------|------------------------------|-----|-------|---------|---------|---------------------|
+| Rochester, NY | 23,29,33,55,41,36,12,67      | 296 | 8     | 12      | 67      | 37                  |
+| New York, NY  | 18,22,26,31,29,40,23,35      | 215 | 8     | 18      | 40      | 28                  |
+
+The total number of custom metrics or timeseries emitted from the `age` distribution metric is **ten (5 x 2)**. For both unique tag value combinations above (Rochester, NY and New York, NY), Datadog stores five timeseries (`sum`,`count`,`min`,`max`, `avg`):
+
+| Metric      | Tags                         |
```

Comment:
This is misleading. A user won't be able to find `<distributionmetric>.avg` in their Metric Summary page with these tags...the timeseries are all associated with one distribution metric. Therefore only the distribution metric is listed in the metric summary page